evennia/utils/batchprocessors.py

```diff
@@ -233,7 +233,7 @@ def read_batchfile(pythonpath, file_ending=".py"):
                 continue
             break
     if not text and decoderr:
-        raise UnicodeDecodeError("\n".join(decoderr))
+        raise UnicodeDecodeError("\n".join(decoderr), bytearray(), 0, 0, '')
```

Comment:
UnicodeDecodeError constructor expects 5 args.

Reply:
Oh? Interesting. I was not aware of this change.